bg-find-sync/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
+ARG  CODE_VERSION=latest
+
+FROM ${DOCKER_REPOSITORY}/${MANAGE_COURSES_BACKEND_IMAGE}:${CODE_VERSION}
```

Comment:
can this not inherit from `batdevcontainerregistry.azurecr.io/manage-courses-backend:latest`

it looks likes to me that we are building a image